test_suite.sh

```diff
@@ -85,6 +85,26 @@ source $TOPDIR/init/bash
 # Reason is that the LMOD cache is normally only updated on the Stratum 0, once everything is ingested
 export LMOD_IGNORE_CACHE=1
 
+software_layer_dir=$(dirname $(realpath $0))
```

Comment:
This whole section up to `echo "test_suite.sh: EESSI_ACCELERATOR_TARGET='${EESSI_ACCELERATOR_TARGET}'"` can now be removed again, now that the EESSI_ACCELERATOR_TARGET is set in `test.sh`, right?